nucliadb_vectors/src/data_point_provider/mod.rs

```diff
@@ -84,12 +85,9 @@ impl Index {
         Ok(())
     }
     fn notify_merger(&self) {
-        use merge_worker::Worker;
-        let notifier = merger::get_notifier();
+        use crate::data_point_provider::merge_worker::Worker;
```

Comment:
Could you move that import on top of the file ? It's definitely an opinionated suggestion but I always feel a bit confuse when the code contains nested imports

Reply:
Done at [d846d76](https://github.com/nuclia/nucliadb/pull/600/commits/d846d76971f64cd8d803c4562c05718265a55ea3)

Reply:
Done at [d846d76](https://github.com/nuclia/nucliadb/pull/600/commits/d846d76971f64cd8d803c4562c05718265a55ea3)